src/workflows-app/utils/method-common.js

```diff
@@ -9,6 +14,37 @@ const MethodSource = Object.freeze({
 const Covid19Methods = ['fetch_sra_to_bam', 'assemble_refbased', 'sarscov2_nextstrain'];
 export const isCovid19Method = (methodName) => Covid19Methods.includes(methodName);
 
+export const submitMethod = async (signal, onDismiss, method, workspace) => {
```

Comment:
clarifying question: is this method used in more than 1 place?

Reply:
Hm, it seems it's only used here. It may be worth moving it within the file that calls it?

Reply:
Yeah if it's only being used once then that makes sense

Reply:
After looking through the code again, this function is actually used in 2 components, so I reverted changes I made that had the `submitMethod` function in a single component.